rest/model/codegen_test.go

```diff
@@ -53,37 +52,6 @@ func TestWords(t *testing.T) {
 	assert.Equal(t, words(s), []string{"this", "is", "a", "field", "name"})
 }
 
-func TestCreateConversionMethods(t *testing.T) {
```

Comment:
Was it necessary to get rid of this test? Can the expected value of the test be modified instead, or this test now covered elsewhere?